qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```
@@ -95,6 +95,18 @@
             iso_label_chk_cmd = 'wmic logicaldisk get VolumeName |findstr -i ${pkg_name}'
             installer_chk_cmd = 'powershell -command "(Get-Item -path "%s:\\virtio-win-guest-tools.exe").VersionInfo.ProductVersion"'
         - driver_install:
+            variants:
+                - with_viorng:
+                    read_rng_cmd = "WIN_UTILS:\\random_%PROCESSOR_ARCHITECTURE%.exe"
+                    driver_test_name = rng
+                - with_viostor:
+                    iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
+                    driver_test_name = iozone
+                    driver_test_params = {'img_size': '${image_size_stg0}'}
+                - with_vioscsi:
+                    iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
+                    driver_test_name = iozone
+                    driver_test_params = {'img_size': '${image_size_stg1}'}
```

They looks duplicated with each other, and why do we exactly need both stg0 and stg1 for all test cases? How about 
```suggestion
    ...
      - @storage:
           images += " stg0"
           image_name_stg0 = "images/stg0"
           image_size_stg0 = 5G
           remove_image_stg0 = yes
           force_create_image_stg0 = yes  
           driver_test_name = iozone
           driver_test_params = {'img_size': '${image_size_stg0}'}
           iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile" 
           variants:
                - with_viostor:
                    drive_format_stg0 = virtio
                - with_vioscsi:
                    drive_format_stg0 = scsi-hd
```